src/js/angular/graphexplore/controllers/graphs-visualizations.controller.js

```diff
@@ -36,6 +37,10 @@ function GraphsVisualizationsCtrl($scope, $rootScope, $repositories, $licenseSer
     $scope.embedded = $location.search().embedded;
     $scope.openedNodeInfoPanel = undefined;
 
+    $scope.invalidLimit = false;
+    $scope.INVALID_LINKS_MSG = 'Invalid links limit';
+    $scope.INVALID_LINKS_TOOLTIP = 'The valid limit range is from 1 - 1000';
```

Comment:
The message better be "The valid limit range is 1-1000" without the "from" or "valid limit range: 1-1000"